src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
```

Comment:
I find your main method to be sufficient to be really good to fulfil all the requirements. May I know why you did it this way instead of creating a new Duke object inside the main class and calling all the methods for that object like I did.